### PR TITLE
Check when ws-edit changes were applied

### PR DESCRIPTION
**Pull request by svor**
The fix for some unstable tests from `org.eclipse.che.selenium.refactor.move`.
Sometimes tests check the state of the project tree before applying refactoring changes. Firstly we need to wait  when all WorkspaceEdit changes were applied.

It is needed for https://github.com/eclipse/che/issues/10399

### CODE REVIEW

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java [hunk 1]
```diff
@@ -106,7 +118,10 @@ public void checkMoveItem0() throws Exception {
     refactor.clickOnExpandIconTree("/src/main/java");
     refactor.chooseDestinationForItem("p1");
     refactor.clickOkButtonRefactorForm();
+    loader.waitOnClosed();
```
Comment (tsmaeder) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java:121]: You put this these in various places. Do you understand when they are needed?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java [hunk 2]
```diff
@@ -106,7 +118,10 @@ public void checkMoveItem0() throws Exception {
...
     refactor.waitMoveItemFormIsClosed();
+    loader.waitOnClosed();
+    events.waitExpectedMessage(APPLY_WORKSPACE_CHANGES, LOAD_PAGE_TIMEOUT_SEC);
```
Comment (tsmaeder) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java]: Is load page timeout the right number here? What would be a reasonable timeout for those refactorings?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveJavaClassToSubpackageTest.java [hunk 3]
```diff
@@ -10,6 +10,8 @@
  */
 package org.eclipse.che.selenium.refactor.move;
 
+import static org.eclipse.che.selenium.core.constant.TestTimeoutsConstants.LOAD_PAGE_TIMEOUT_SEC;
```
Comment (tsmaeder) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveJavaClassToSubpackageTest.java]: Same as above, is this the right timeout?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java (hunk 1 above)
Comment (svor) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java:121]: it is needed in places where UI loader can be shown

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java (hunk 2 above)
Comment (svor) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/move/MoveItemsTest.java]: I've changed to `DEFAULT_TIMEOUT`, the value is the same - 10 sec. It should be enough to apply changes.

**Review by tsmaeder — APPROVED** [review]

**Pull request merged**